MODULE.bazel

```
@@ -768,7 +768,7 @@ cc_library(
 )
 
 http_archive(
-    name = "bitcoin_core",
+    name = "bitcoin_core_linux",
```

```suggestion
    name = "bitcoin_core_linux_x86",
```

would suggest adding the arch to keep it consistent with the darwin ones